src/journal-remote/journal-upload.h

```diff
@@ -49,10 +55,16 @@ typedef struct Uploader {
         /* general metrics */
         const char *state_file;
 
+        double bytes_left;
+        double entries_left;
```

Comment:
sorry, but no way. we are not going to count bytes in floating point. use uint64_t or so.